Quadriga/src/main/java/edu/asu/spring/quadriga/service/impl/transformation/TransformationSaveService.java

```diff
@@ -0,0 +1,35 @@
+package edu.asu.spring.quadriga.service.impl.transformation;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+
+import edu.asu.spring.quadriga.domain.workspace.ITransformationFile;
+import edu.asu.spring.quadriga.exceptions.FileStorageException;
+import edu.asu.spring.quadriga.service.transformation.ITransformationSaveService;
+import edu.asu.spring.quadriga.utilities.IFileSaveUtility;
+
+/**
+ * This service contains method to save contents of transformation files to
+ * local file storage
+ * 
+ * @author yoganandakishore
+ *
+ */
+
+@Service
+public class TransformationSaveService implements ITransformationSaveService {
+
+    @Qualifier("transformfileSaveUtil")
+    @Autowired
+    private IFileSaveUtility transformationFileManager;
+
+    @Override
+    public boolean saveFileToLocal(ITransformationFile transformationFile) throws FileStorageException {
+
+        return (transformationFileManager.saveFiletoDir(transformationFile.getPatternTitle(),
```

Comment:
also, my old comment still applies:
"what about the metadata file? that contains some rudimentary info about the transformation (e.g. title and description)"